### PR TITLE
ADD: emotion API

### PR DESCRIPTION
**Pull request by madisonmay**

### CODE REVIEW

File: lib/indico.rb [hunk 1]
```diff
@@ -57,19 +57,23 @@ def self.language(text, config = nil)
...
-  def self.keywords(text, config = nil)
-    unless !config or config.key?(:v) or config.key?(:version)
-      config[:version] = "2"
-    end
-    if config and config.key?(:language) and config[:language] != "english"
-      config[:version] = "1"
-    end
-    api_handler(text, 'keywords', config)
-
+  def self.keywords(text, config = nil)
```
Comment (madisonmay) [inline lib/indico.rb]: Ignore this it's the difference between carriage returns and newline characters

File: spec/indico_batch_spec.rb [hunk 2]
```diff
@@ -21,6 +21,15 @@
     expect(Set.new(response[1].keys)).to eql(expected_keys)
   end
 
+  it 'should tag text with correct emotion tags' do
```
Comment (anneCarlson) [inline spec/indico_batch_spec.rb]: Are there any optional params we should be checking? Otherwise, looks good!

**Comment by anneCarlson** [comment]
Good to merge if it doesn't take any optional params, otherwise, needs a test for those.

File: spec/indico_batch_spec.rb (hunk 2 above)
Comment (madisonmay) [inline spec/indico_batch_spec.rb]: top_n and threshold apply

Comment (madisonmay) [inline spec/indico_batch_spec.rb]: However in general I feel weird having this being the responsibility of the client libs

Comment (madisonmay) [inline spec/indico_batch_spec.rb]: We test all of that on the server -- all we would really be testing is if the keyword arguments are passed along, which is independent of any particular API

**Pull request merged**